tests/rptest/redpanda_cloud_tests/high_throughput_test.py

```diff
@@ -227,39 +246,72 @@ def __init__(self, test_ctx: TestContext, *args, **kwargs):
                              node_prealloc_count=3,
                              num_brokers=num_brokers,
                              extra_rp_conf=extra_rp_conf,
-                             cloud_tier=cloud_tier,
+                             cloud_tier=self.config_profile_name,
                              disable_cloud_storage_diagnostics=True,
                              **kwargs)
 
-        self.tier_config = self.redpanda.advertised_tier_config
-        if cloud_tier == CloudTierName.DOCKER:
+        install_pack_client = InstallPackClient(
+            self.redpanda._cloud_cluster.config.install_pack_url_template,
+            self.redpanda._cloud_cluster.config.install_pack_auth_type,
+            self.redpanda._cloud_cluster.config.install_pack_auth)
+        install_pack_version = self.redpanda._cloud_cluster.config.install_pack_ver
+        install_pack = install_pack_client.getInstallPack(install_pack_version)
+        config_profile = install_pack['config_profiles'][
+            self.config_profile_name]
+        cluster_config = config_profile['cluster_config']
+
+        self._num_brokers = config_profile['nodes_count']
+        self._segment_size = cluster_config['log_segment_size']
```

Comment:
from conversation with alex: i'll change this value used in the tests to be 0.5 * the `log_segment_size`